doc/source/change_proposal_process.rst

```diff
@@ -152,36 +154,59 @@ These steps are necessary to initialize a project repo enabling you to contribut
     before the next “ERROR” line. Depending on the type of the error,
     the important information might be in another line, though.
 
-19. To prevent cycling over tiny bugs repeatedly, it might be helpful
+20. To prevent cycling over tiny bugs repeatedly, it might be helpful
     to test a changeset locally first before pushing it to GitHub. We
     mentioned this in step 7 before.
 
-20. Once you found the cause of the failed check, return to your local
+21. Once you found the cause of the failed check, return to your local
     repository, and fix it. You may close the Zuul browser tab and
     return to step 9.
+    
+Manual review and approval process
+==================================
 
-21. In the meantime, the manual review phase has started in
+22. In the meantime, the manual review phase has started in
     parallel. You may either hope that some peer monitors the project
-    and comments on the PR or copy the link in a messenger so that
-    potential reviewers notice. There is no technical requirement for
+    and comments on the PR or you copy the link in a messenger so that
+    potential reviewers notice.
+    
+23. To see the results of the pending change you may take a look at the preliminary version.
+    First locate the text "all checks have passed" next to the green checkmark
+    
+24. Click on "Show all checks" and then on "Details" below. 
+    Under the headline "Summary" all build jobs are listed. 
+    Identify the one with "tox-docs" in it and click on the link. 
+    Select "Artifacts" and click on "Docs preview site".
+    Now the rendered site is displayed.    
+    
+25. There is no technical requirement for
     reviews but having at least two other colleagues verifying your
     change set is considered good practice. However, every team may
-    define its own review policies. Regardless of this policy nobody
+    define its own [review policies](xxx). Regardless of this policy nobody
     can merge directly into the repository, since Zuul prevents
     this. Only Zuul has effective write permissions to it.
 
-22. Watch the comments on the PR. It is your responsibility to either
+26. Watch the comments on the PR. It is your responsibility to either
     fix (or reject) the comments of your peers, not theirs. If you
     need to change texts, go back to step 9, create new commits, and
     push them to the same branch of the repository. They will appear
     in the same PR.
 
-23. If the project agrees that the change set is ready to be merged,
+27. If the project agrees that the change set is ready to be merged,
     someone has to put the label “gate” to the commit. To do so, open
     the PR, select the “Conversation” tab, and locate the “Labels”
     section on the right-hand sidebar. Click on the tiny wheel and
-    select the “gate” label.
+    select the “gate” label. Each team can define its own [approval policy](xxx). 
```

Comment:
let us please drop it from here. Approving/merging is not part of the change_proposal itself and people should not even know how it works in detail

Reply:
ok I agree with you. That´s a good point should I paste the rest into the approver.rst document for later use?

Reply:
yeah, canbbe done